spec/rut-helpers.spec.js

```diff
@@ -4,15 +4,15 @@ describe('rutHelper', () => {
   describe('rutHelper.rutValidate', () => {
     it('should return true if string is a valid rut', () => {
       expect(rutHelpers.rutValidate('7.618.285-K')).toBe(true);
+      expect(rutHelpers.rutValidate('1-9')).toBe(true);
     });
 
     it('should return true if string is a valid rut without punctuation', () => {
       expect(rutHelpers.rutValidate('7618285K')).toBe(true);
     });
 
-    it('should return false if string is not a valid rut', () => {
-      expect(rutHelpers.rutValidate('7.618.212-K')).toBe(false);
-      expect(rutHelpers.rutValidate('7618212K')).toBe(false);
+    it('should return false if string is a valid rut with extra characters', () => {
+      expect(rutHelpers.rutValidate('7618285K1K')).toBe(false);
```

Comment:
I think this test is only passing because the 1 between the two K makes the rut invalid, not because of the new `match`. In fact I think that, given the new behavior in `rutClean`, `rutValidate` would never return `false` because of the new check